### PR TITLE
change DIMet version to 0.2.4

### PR DESCRIPTION
**Pull request by bdartigues**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

### CODE REVIEW

**Comment by bdartigues** [comment]
Need to commit more changes to validate the pull request

File: tools/dimet/test-data/legend.svg
Comment (bgruening) [inline tools/dimet/test-data/legend.svg:1]: This result looks wired, is that expected?

**Review by bgruening — APPROVED** [review]
Thanks! Just one question :)

File: tools/dimet/test-data/legend.svg
Comment (bdartigues) [reply tools/dimet/test-data/legend.svg:1]: It is the seaborn update that has produced those modifications of the figures, and they are now even better than before. Thank you for asking :)

**Pull request merged**